<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QMainWindow" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1137</width>
    <height>916</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM Vizualizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
	border-radius:15px;
	font-family:century gohtic;
}

QFrame[accessibleName=&quot;bodyFrame&quot;]{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(191, 186, 181, 255), stop:1 rgba(231, 227, 223, 255));
}

QFrame[accessibleName=&quot;mainFrame&quot;]{
	background-color:rgb(244, 244, 244);
}

QFrame[accessibleName=&quot;titleFrame&quot;]{
	background-color:rgb(244, 244, 244);
}

QLabel[accessibleName=&quot;titleLabel&quot;]{
	font-size:40px;
	margin:10px;
	font-weight:bold;
	color:rgb(0, 51, 69);
}

QLabel[accessibleName=&quot;feature1Label&quot;],QLabel[accessibleName=&quot;feature2Label&quot;]{
	font-size:18px;
	margin:10px;
	font-weight:bold;
	color:rgb(0, 51, 69);
}

QLabel[accessibleName=&quot;numeFisierLabel&quot;]{
	font-size:40px;
	margin:10px;
	font-weight:bold;
	background-color:rgb(27, 134, 151);
	color:rgb(200, 200, 200);
	padding:5px;
}

QFrame[accessibleName=&quot;rightFrame&quot;],QFrame[accessibleName=&quot;leftFrame&quot;]{
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(191, 186, 181, 255), stop:1 rgba(231, 227, 223, 255));;
}

QFrame[accessibleName=&quot;leftFrameTop&quot;],QFrame[accessibleName=&quot;leftFrameButtom&quot;]{
	background-color:rgb(244, 244, 244);
}

QPushButton[accessibleName=&quot;importFisierButton&quot;],QPushButton[accessibleName=&quot;loadDataButton&quot;]{
	background:rgb(27, 134, 151);
	padding:10px;
	font-size:30px;
	margin-bottom:10px;
	margin:5%;
	color:white;
}

QPushButton[accessibleName=&quot;importFisierButton&quot;]:hover,QPushButton[accessibleName=&quot;loadDataButton&quot;]:hover{
	background:rgb(15, 100, 117);
	padding:10px;
	font-size:30px;
	margin:10%;
	color:white;
}

QPushButton[accessibleName=&quot;dataSet1Button&quot;],QPushButton[accessibleName=&quot;dataSet2Button&quot;],QPushButton[accessibleName=&quot;dataSet3Button&quot;],QPushButton[accessibleName=&quot;dataSet4Button&quot;],QPushButton[accessibleName=&quot;randomDataButton&quot;]{
	background:rgb(27, 134, 151);
	padding:15%;
	padding-top:20%;
	padding-bottom:20%;
	font-size:25px;
	margin: 5%;
	color:white;
}

QPushButton[accessibleName=&quot;dataSet1Button&quot;]:hover,QPushButton[accessibleName=&quot;dataSet2Button&quot;]:hover,QPushButton[accessibleName=&quot;dataSet3Button&quot;]:hover,QPushButton[accessibleName=&quot;dataSet4Button&quot;]:hover{
	background:rgb(15, 100, 117);
	font-size:25px;
	padding:10%;
	margin: 1%;
	color:white;
}

QPushButton[accessibleName=&quot;startSVMButton&quot;]{
	background:rgb(27, 134, 151);
	padding:10px;
	font-size:30px;
	margin-right:100%;
	margin-left:100%;
	margin-bottom:10px;
	color:white;
}

QPushButton[accessibleName=&quot;startSVMButton&quot;]:hover{
	background:rgb(15, 100, 117);
	padding:10px;
	font-size:30px;
	margin-right:80%;
	margin-left:80%;
	color:white;
}

QWidget[accessibleName=&quot;plotWidget&quot;]{
	background-color:rgb(244, 244, 244);
}

QComboBox {
	border:none;
	font-size:18px;
	padding:10%;
	padding-left:15%;
	margin-left:5%;
	margin-right:5%;
	color:rgb(102, 132, 143);
}

QComboBox::down-arrow {
	image: url(C:/Users/Darius/Desktop/Qt Designer Apps/down-arrow.png)
}

QComboBox::drop-down:button{
	border-radius:5px; 
	background:transparent;
	margin-right:20%;	
}

QComboBox QAbstractItemView{
	background-color:white;
	selection-background-color:rgb(27, 134, 151);
	selection-color:white;
	shadow:none;
	padding:10px;
	box-shadow:none;
	margin-left:5px;
	margin-right:5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QFrame" name="bodyFrame">
      <property name="accessibleName">
       <string>bodyFrame</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="titleFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>titleFrame</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="accessibleName">
             <string>titleLabel</string>
            </property>
            <property name="text">
             <string>SVM Visualizer</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mainFrame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>mainFrame</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="leftFrame">
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="accessibleName">
             <string>leftFrame</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QFrame" name="leftFrameTop">
               <property name="accessibleName">
                <string>leftFrameTop</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="numeFisierLabel">
                  <property name="accessibleName">
                   <string>numeFisierLabel</string>
                  </property>
                  <property name="layoutDirection">
                   <enum>Qt::LeftToRight</enum>
                  </property>
                  <property name="text">
                   <string>No file chosen...</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feature1Label">
                  <property name="accessibleName">
                   <string>feature1Label</string>
                  </property>
                  <property name="text">
                   <string>Feature 1:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="feature1ComboBox">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>feature1ComboBox</string>
                  </property>
                  <property name="currentText">
                   <string>one</string>
                  </property>
                  <property name="placeholderText">
                   <string/>
                  </property>
                  <item>
                   <property name="text">
                    <string>one</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>two</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>three</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="feature2Label">
                  <property name="accessibleName">
                   <string>feature2Label</string>
                  </property>
                  <property name="text">
                   <string>Feature 2:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="feature2ComboBox">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>feature2ComboBox</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Feature 2</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QPushButton" name="importFisierButton">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="accessibleName">
                      <string>importFisierButton</string>
                     </property>
                     <property name="text">
                      <string>File Import</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="loadDataButton">
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="accessibleName">
                      <string>loadDataButton</string>
                     </property>
                     <property name="text">
                      <string>Load Data</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="leftFrameBottom">
               <property name="accessibleName">
                <string>leftFrameButtom</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QPushButton" name="dataSet1Button">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>dataSet1Button</string>
                  </property>
                  <property name="text">
                   <string>DataSet1</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QPushButton" name="dataSet2Button">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>dataSet2Button</string>
                  </property>
                  <property name="text">
                   <string>DataSet2</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="dataSet4Button">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>dataSet4Button</string>
                  </property>
                  <property name="text">
                   <string>DataSet4</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="dataSet3Button">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>dataSet3Button</string>
                  </property>
                  <property name="text">
                   <string>DataSet3</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QPushButton" name="randomDataButton">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="accessibleName">
                   <string>randomDataButton</string>
                  </property>
                  <property name="text">
                   <string>PushButton</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="rightFrame">
            <property name="accessibleName">
             <string>rightFrame</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QPushButton" name="startSVMButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="accessibleName">
                <string>startSVMButton</string>
               </property>
               <property name="text">
                <string>START</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QFrame" name="plotFrame">
               <property name="accessibleName">
                <string>plotFrame</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_5"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
